### DISCUSSION TITLE
Documentation Availability

### DISCUSSION THREAD

**Discussion by BigFriendly** [cubewise-code/tm1py#787] [General]
Hi,

For the last few weeks, I've been unable to find populated documentation at the referenced site: 

https://tm1py.readthedocs.io/en/latest/api.html

I see the left navigation menu with "TM1 Services", "TM1 Objects", etc, but there is no content when clicked.  Could someone please advise?

Thanks,
-- John

**Comment by MariusWirtz** [comment]
Hi @BigFriendly, 

the documentation on readthedocs.org gets generated automatically based on the docstrings in the code. Currently, due to a dependency issue between mdxpy and tm1py, it doesn't build. We will probably fix it this week.

In the meantime, feel free to post a question in the discussions.

Marius

**Reply by BigFriendly** [reply]
Glad to hear it's on the list to be fixed.  It's been sorely missed!

Thanks.

**Reply by MariusWirtz** [reply]
Documentation is back
https://tm1py.readthedocs.io/en/master/index.html
